google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/GenerateClirrIgnoredDifferences.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable;
+
+import com.google.api.core.InternalApi;
+import com.google.cloud.bigtable.data.v2.BigtableDataClient;
+import com.google.common.base.Predicate;
+import com.google.common.base.Predicates;
+import com.google.common.collect.Collections2;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.reflect.ClassPath;
+import com.google.common.reflect.ClassPath.ClassInfo;
+import java.io.IOException;
+import java.util.Collection;
+import org.checkerframework.checker.nullness.compatqual.NullableDecl;
+
+/** Internal helper to update clirr-ignored-differences */
+@InternalApi
```

Comment:
I don't think this should be included in source code and published. I think https://github.com/googleapis/java-bigtable/tree/master/scripts could be a better spot? Or a util module?